### PR TITLE
[cityhash] Add new port

### PR DESCRIPTION
**Pull request by zi-m**
https://github.com/google/cityhash

### CODE REVIEW

**Comment by zi-m** [comment]
> @zi-m is it feasible to be able to configure the config.h file on the fly?

I have done it for linux, but not for windows ((

**Comment by vicroms** [comment]
This PR looks good to me, we have other ports that ship a `config.h` for Windows.

The `usage` file is not required. Instructions to link the libraries using CMake are auto-generated by **vcpkg** when config files are detected.

File: ports/cityhash/usage [hunk 1]
```diff
@@ -0,0 +1,4 @@
+The package cityhash provides CMake targets:
```
Comment (vicroms) [inline ports/cityhash/usage]: This file is not needed, **vcpkg** can auto-generate this instructions.

**Review by vicroms — CHANGES_REQUESTED** [review]

File: ports/cityhash/usage (hunk 1 above)
Comment (zi-m) [reply ports/cityhash/usage]: done! I have just removed file.

**Comment by vicroms** [comment]
Thanks for the PR!

**Review by vicroms — APPROVED** [review]

**Pull request merged**